projects/sbb-esta/angular-business/chip/src/chip-input/chip-input.component.ts

```diff
@@ -0,0 +1,260 @@
+      this.ngControl.valueAccessor = this;
+    }
+
+    // Force setter to be called in case id was not specified.
+    this.id = this.id;
+  }
+
+  ngOnInit(): void {
+    this.selectionModel = new SelectionModel<string>(true);
+    this._initializeSelection();
+    if (this.autocomplete) {
+      this.autocomplete.optionSelected.subscribe(event => this.onSelect(event.option.value));
+    }
+  }
+
+  /**
+   * Sets the chip input's value. Part of the ControlValueAccessor interface
+   * required to integrate with Angular's core forms API.
+   *
+   * @param value New value to be written to the model.
+   */
+  writeValue(value: string[]): void {
+    if (this.selectionModel) {
+      value.forEach(v => this.onSelect(v));
+    }
+  }
+
+  /**
+   * Disables the chip input. Part of the ControlValueAccessor interface required
+   * to integrate with Angular's core forms API.
+   *
+   * @param isDisabled Sets whether the component is disabled.
+   */
+  setDisabledState(isDisabled: boolean): void {
+    this.disabled = isDisabled;
+    this._changeDetectorRef.markForCheck();
+    this.stateChanges.next();
+  }
+
+  /**
+   * Adds a given value to the current selected values.
+   */
+  onSelect(option: string) {
+    if (!this.selectionModel.isSelected(option)) {
+      this.selectionModel.select(option);
+      this._propagateChanges();
+    }
+    this.inputElement.nativeElement.value = '';
+  }
+
+  /**
+   * Removes a given value from the current selected values.
+   */
+  deselectOption(option: string) {
+    if (this.selectionModel.isSelected(option)) {
+      this.selectionModel.deselect(option);
+      this._propagateChanges(option);
+    }
+  }
+
+  /** `View -> model callback called when value changes` */
+  onChange: (value: any) => void = () => {};
+
+  /**
+   * Saves a callback function to be invoked when the chip input's value
+   * changes from user input. Part of the ControlValueAccessor interface
+   * required to integrate with Angular's core forms API.
+   *
+   * @param fn Callback to be triggered when the value changes.
+   */
+  registerOnChange(fn: any): void {
+    this._onChangeCallback = fn;
+  }
+
+  /**
+   * Saves a callback function to be invoked when the chip input is blurred
+   * by the user. Part of the ControlValueAccessor interface required
+   * to integrate with Angular's core forms API.
+   *
+   * @param fn Callback to be triggered when the component has been touched.
+   */
+  registerOnTouched(fn: any): void {
+    this._onTouchedCallback = fn;
+  }
+
+  /**
+   * Implemented as part of FormFieldControl.
+   * @docs-private
+   */
+  setDescribedByIds(ids: string[]): void {
+    this._ariaDescribedby = ids.join(' ');
+  }
+
+  /** Emits change event to set the model value. */
+  private _propagateChanges(fallbackValue?: string): void {
+    this._value = this.selectionModel.selected;
+    this.onChange(this._value);
+    this.valueChange.emit(new SbbChipInputChange(this, this._value));
```

Comment:
valueChange should just emit `new SbbChipInputChange(this, this._value)`.
Remove the following two lines with `this._value ? ...`
(Also `this._value` is an array and even empty would never equal to false)